docs/docRoot.js

```diff
@@ -148,33 +147,30 @@ const apiDocumentation = {
         '/cities': {
             'get': getAllCitiesSwagger,
         },
-        '/cities/{cityId}/listings': {
```

Comment:
since the doc is common for all the versions, wouldn't this change make it impossible to test the APIs of older versions?

Reply:
True, Will update to show these only for v2 versions